### ISSUE TITLE
ocrmypdf package for M1

### ISSUE THREAD

**Issue by xave** [NixOS/nixpkgs#145028]
**Project description**
`ocrmypdf` for M1. The package on nix does not build on M1 with `nix-env -i ocrmypdf`. Nix suggests exporting NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1. This also does not resolve the issue.
**Metadata**

* homepage URL: https://ocrmypdf.readthedocs.io/en/latest/index.html
* source URL: https://github.com/jbarlow83/OCRmyPDF
* license: mit, bsd, gpl2+ , ...
* platforms: Debian, Ubuntu, Windows Subsystem for Linux, Fedora, macOS

**Comment by Artturin** [comment]
Logs?

**Comment by xave** [comment]
```installing 'ocrmypdf-12.6.0'
**error**: Package ‘python3.9-ocrmypdf-12.6.0’ in /nix/store/pkmxs4v1rxmz2mhwpzll4ajmpaavgk1s-nixpkgs-21.11pre325387.f112b512e17/nixpkgs/pkgs/development/python-modules/ocrmypdf/default.nix:90 is not supported on ‘aarch64-darwin’, refusing to evaluate.```

**Comment by Artturin** [comment]
Build log

**Comment by xave** [comment]
What is the exact command?

**Comment by dotlambda** [comment]
> What is the exact command?

You should run `NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1 nix-env -i ocrmypdf` and paste the output.

**Comment by xave** [comment]
Hi. It would be more helpful if one of you has a command to automatically capture the output.

What I would typically do is run `command >> log.txt`. That isn't working with nix-env. You both are more knowledgable about nix than I am. How would you capture the log in a simple command?

**Comment by Artturin** [comment]
```command 2>&1 | tee log``` this works for pretty much everything afaik

**Comment by xave** [comment]
Thank you. Here is the log.
[ocrmypdf-log.txt](https://github.com/NixOS/nixpkgs/files/7529097/ocrmypdf-log.txt)

**Comment by Artturin** [comment]
tesseract build fails but https://github.com/tesseract-ocr/tesseract/issues/3447 its fixed and we need to update to 4.1.2

**Comment by Artturin** [comment]
use nixpkgs-review to review #145654
https://github.com/Mic92/nixpkgs-review

```nixpkgs-review pr 145654 -p ocrmypdf```
